### ISSUE TITLE
proposal: Consolidate success outputs

### ISSUE THREAD

**Issue by jpreese** [instrumenta/conftest#246]
# Current Behavior

Currently when running `conftest test` each rule that passess successfully is output to the console window. This doesn't provide much value when many of the rules will be called `warn` / `deny` / `violation`

![image](https://user-images.githubusercontent.com/6980197/74840878-bf68c100-52f5-11ea-8676-cb250089b728.png)

Similar issues have been called out (https://github.com/instrumenta/conftest/issues/190), and while I disagree with a failure-only type mode, I think there's value in addressing the problem.

# Proposal

In the same thread, I tossed out the idea of mimic'ing `go test` (https://github.com/instrumenta/conftest/issues/190#issuecomment-566540397).

The idea would be that `conftest test`, by default, would say `PASS` once per package/rule name. e.g.

```
PASS - data.policy.deny
```

_(optionally we could include how many rules of that name we processed, if we thought it was valuable)_

In the same proposal, we could then output when no rules were processed (again similar to `go test`)

```
?       github.com/instrumenta/conftest [no test files]
```

Which would address https://github.com/instrumenta/conftest/issues/181

**Comment by Blokje5** [comment]
I like the idea. But what would happen in case of a failure. E.g. we have a package with 3 deny rules and one of them fails? Do we then move back to displaying two passes and one failure? 

I think it is a good idea to include counts of rules processed from the start, it would make keeping track of these situations easier. I like going in the direction of `go test` and also using an `?` to indicate no rules processed.

**Comment by jpreese** [comment]
I was just thinking to leave the failures as is. So in this scenario we would say

```
PASS - data.policy.warn 2/3
FAIL - Found deployment hello-kubernetes but deployments are not allowed
```

Or we could just hide successes altogether when a failure occurs (similar to go test)

```
FAIL - Found deployment hello-kubernetes but deployments are not allowed
```

**Comment by Blokje5** [comment]
Closed by #254
